addon/serializers/github-repository.js

```diff
@@ -7,6 +7,13 @@ export default GithubSerializer.extend({
       fullName: hash.full_name,
       name: hash.name,
       description: hash.description,
+      url: hash.html_url,
+      language: hash.language,
+      isFork: hash.fork,
+      isPrivate: hash.private,
```

Comment:
I think I would like to stick to the GitHub naming conventions after all.  If you have time to make the change then go for it.  Otherwise then I'll merge this in tonight, make the changes, add the tests and release.  Thanks!